DQMOffline/Trigger/python/HILowLumiHLTOfflineSource_cfi.py

```diff
@@ -0,0 +1,151 @@
+import FWCore.ParameterSet.Config as cms
+import math
+
```

Comment:
Ah, this was mostly based on the FSQ DQM since there is some overlap in trigger types. They used it for abs(), but I don't think it is used here. Slipped thru the edits